.travis.yml

```diff
@@ -35,14 +35,18 @@ matrix:
       compiler: clang
       env:
         SCAN_BUILD=yes
-        CMAKE_FLAGS=-DGOOGLE_CLOUD_CPP_GRPC_PROVIDER=package
-        DISTRO=ubuntu-with-dependencies
+        DISTRO=ubuntu
         DISTRO_VERSION=16.04
+      # Because this builds installs gRPC we can skip that submodule, that
+      # saves substantial time in the build, but requires some tweaking of the
+      # installation step.
+      git:
+        submodules: false
       install:
-        - # First build the usual Ubuntu image, then use that to build an image
-          # with gRPC already compiled and installed.
-        - DISTRO=ubuntu SCAN_BUILD=no ./ci/install-linux.sh
-        - ./ci/install-linux.sh
+        - git submodule update --init
+                    third_party/googleapis
```

Comment:
4 space indent? This is like 8, 9 space indent? Similarly below and elsewhere.

Reply:
Fixed.  4 from the beginning for `git`, let me know if you prefer 4 from `- `.